File: problemsets/Codeforces/Python/A1077.py
"""
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 """
for _ in '0'*int(input()):
  a,b,k=map(int,input().split())
  print(k//2*(a-b)+k%2*a)

File: problemsets/Codeforces/Java/A1077.java
/**
 *
 * Author:  Juarez Paulino(coderemite)
 * Email: juarez.paulino@gmail.com
 *
 */
import java.util.*;

public class A1077 {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int t = in.nextInt();
		while (t-->0) {
			long a = in.nextInt(), b = in.nextInt(), k = in.nextInt();
			System.out.println(k/2*(a-b) + k%2*a);
		}
	}
}